mitype/app.py

```diff
@@ -197,13 +197,16 @@ def key_printer(self, win, key):
         elif mitype.keycheck.is_backspace(key):
             self.erase_key()
 
-        # Check for space
-        elif key == " ":
-            self.check_word()
```

Comment:
We can pobably just change this to -
```
elif key == " " and len(self.current_word) == 0:
    self.check_word()
```

Reply:
Also, `self.current_word = ""` instead of `len(self.current_word ) == 0` looks good

Reply:
I think we still need the check in both if statements, not just the one for `self.check_word()`. `self.current_word == ""` sounds fine though.

Reply:
Right. Just tested out doesn't work with above suggestion.
Something like this should work though.
```
        # Ignore spaces at the start of the word (Plover support)
        elif key==" ":
            if self.current_word!="":
                self.check_word()

        elif mitype.keycheck.is_valid_initial_key(key):
            self.appendkey(key)
```

Reply:
Yeah that's better and still works.